shared/common-adapters/oriented-image.types.js.flow

```diff
@@ -5,4 +5,5 @@ export type Props = {
   src: string,
   style?: StylesDesktop,
   onLoad: () => void,
+  preview?: boolean,
```

Comment:
maybe this is more accurate to say localFile or something

Reply:
Updated!